string dest
nav_msgs/OccupancyGrid grid
---
int32 response